### ISSUE TITLE
Cheat discovery queue

### ISSUE THREAD

**Issue by Revadike** [tfedor/AugmentedSteam#426]
Implement the featured from https://github.com/PotcFdk/SteamDiscoveryQueueAutoSkipper by @PotcFdk

**Comment by tfedor** [comment]
Only if author wants us to, or if we have his permission

**Comment by Revadike** [comment]
Sure, hence I tagged him. Nevertheless, it's only an example. There are more discovery queue cheat scripts out there and I don't think I'll have a problem writing one from scratch either.

**Comment by MxtOUT** [comment]
But should that be a permanent feature of AS?
I think it's rather useless outside of these events and even then you're probably better off using a dedicated script for that

**Comment by Revadike** [comment]
> But should that be a permanent feature of AS?
> I think it's rather useless outside of these events 
So what?

> and even then you're probably better off using a dedicated script for that
Why have many scripts when you can have everything available in AS?

**Comment by PotcFdk** [comment]
Hi, I mean, there shouldn't be any big issues. My script is currently Apache v2 licensed, which is extremely permissive, and your project is GPLv3. FSF considers this direction to be A-OK (so it's possible to combine Apache v2 + GPLv3 code into a GPLv3 project).

Of course the script itself would stay Apache v2 and thus allow you to make changes to it (in form of a fork) that could carry a non-Apache license (most likely GPLv3). In case of the GPL I couldn't merge back those changes unless all contributors that had to do with those changes agree to have it merged upstream with the Apache license.
Alternatively, I might consider relicensing to GPLv3 myself - it's not like it would make much of a difference, seeing as it's a relatively small project.
I would still like to avoid an unnecessary fork - it's a small project already, so there's no use splitting up the user base even more. If improvements exist, they should reach everyone, not just _some_ random users.

**tl;dr**: You can legally include this stuff into your project even without asking me, but if you plan to actually do it, please keep me updated so that we can make it easy for everyone involved.
